legacy/utils.js

```diff
@@ -494,3 +495,34 @@ async function callAsposeToExport({
   req.write(JSON.stringify(data));
   req.end();
 }
+
+// returns a list of the lowest level variant values' ids (i.e. SKUs) from a list of variant values
+async function getLowestVariantValuesList(valuesList, namespace) {
+  const helper = new PimExportHelper(namespace);
+  let parentValueLengthMap = new Map();
```

Comment:
you can declare multiple variables in the same 'line;, i.e.
```
let parentValueLengthMap = new Map(),
    numOfParentValues,
    highestNumOfParentValues = 0,
    parentValues,
    vvId;
```
Take note of the `;` and `,`.